pkg/client/linstor.go

```diff
@@ -730,6 +730,33 @@ func (s *Linstor) CapacityBytes(ctx context.Context, storagePool string, segment
 		return 0, fmt.Errorf("unable to get capacity: %w", err)
 	}
 
+	allNodes, err := s.client.Nodes.GetAll(ctx, &lapi.ListOpts{Cached: &cached})
+	if err != nil {
+		return 0, fmt.Errorf("unable to get nodes: %w", err)
+	}
+
+	var filteredNodes []string
+
+	for i := range allNodes {
+		node := &allNodes[i]
+
+		if !slice.ContainsString(requestedNodes, node.Name) {
+			continue
+		}
+
+		if slice.ContainsString(node.Flags, lapiconsts.FlagEvicted) || slice.ContainsString(node.Flags, lapiconsts.FlagEvacuate) {
+			continue
+		}
+
+		if node.Props[lapiconsts.KeyAutoplaceAllowTarget] == "false" {
```

Comment:
not in this commit, but if we have that more often some helper that gets the slice and a key and checks for "does it exist" and if yes, is it false (for whatever string version of false) might make sense.